src/lib/phdi-building-blocks/phdi_building_blocks/fhir.py

```diff
@@ -24,6 +24,7 @@ def generate_filename(blob_name: str, message_index: int) -> str:
 class AzureFhirserverCredentialManager:
     """Manager for handling Azure credentials for access to the FHIR server"""
 
+    # TODO: Generalize this class to decouple from Azure
```

Comment:
This is _super_ nitpicky, but can we move this comment to be above the class (i.e. line 23)?

Reply:
Done.